packages/x-components/src/x-modules/history-queries/components/history-queries.vue

```diff
@@ -50,6 +50,7 @@
   import BaseSuggestions from '../../../components/suggestions/base-suggestions.vue';
   import { Getter } from '../../../components/decorators/store.decorators';
   import { xComponentMixin } from '../../../components/x-component.mixin';
+  import SearchInput from '../../search-box/components/search-input.vue';
```

Comment:
Remove this

Reply:
Wow! You are right, i think (hope) that IDE added this. I am removing it.